src/App.tsx

```
@@ -195,41 +194,71 @@ const WrappedMenuPage = () => {
-      encoded={params.getAll("encoded")}
-      onChangeCompared={
-        // Update the page query params
-        (encoded) => {
-          setParams({ encoded });
-        }
-      }
+      encoded={encodedList}
+      showComparison={showComparison}
+      onChangeCompared={(encoded) => {
+        const newParams = new URLSearchParams();
+        encoded.forEach((e) => newParams.append("encoded", e));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)